cmd/jujud/agent/unit/manifolds.go

```diff
@@ -233,6 +233,7 @@ func Manifolds(config ManifoldsConfig) dependency.Manifolds {
 			AgentName:         agentName,
 			APICallerName:     apiCallerName,
 			FortressName:      migrationFortressName,
+			Clock:             clock.WallClock,
```

Comment:
There should be a clock passed in as monifold config to use.

Reply:
Not in the unit agent, but there are for the machine and caasoperator manifolds. I've changed those - do you want me to introduce one for the unit agent?

Reply:
Not at this stage. Unless there is at least one other place that uses a wall clock.

Reply:
Just looked, there are three other clock.WallClock usages in that file. We should probably introduce a clock in the config like the others.